README.md

```
@@ -158,6 +158,64 @@ users[2]{id,name,role}:
+```php
+echo Toon::encode([
+    'forecast' => [
+        ['day' => 'Mon', 'temp' => ['min' => -2, 'max' => 4], 'condition' => 'snow'],
+        ['day' => 'Tue', 'temp' => ['min' => 1, 'max' => 7], 'condition' => 'cloudy'],
+    ]
+]);
+```
+
+Output:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)